### PR TITLE
Create MVP deployment files

### PR DESCRIPTION
**Pull request by pjbgf**
- Add the yaml files required to deploy the operator.
- Add Dockerfile.
- Create Usage.md to explain current features and how to install the operator.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM golang:1.14-alpine AS build
+WORKDIR /go/src/saschagrunert/seccomp-operator
+RUN apk --no-cache add build-base git bzr mercurial gcc
+
+# Speed up build by leveraging docker layer caching
+COPY go.mod go.sum ./
+RUN go mod download
+
+ADD . /go/src/saschagrunert/seccomp-operator
+RUN go build -o /seccomp-operator
```
Comment (saschagrunert) [inline Dockerfile:10]:
```suggestion
RUN go build -ldflags '-s -w' -o /seccomp-operator
```

File: deploy/operator.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: Deployment
```
Comment (saschagrunert) [inline deploy/operator.yaml:2]: The operator should be a DaemonSet, right?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+
+
+FROM alpine
```
Comment (saschagrunert) [inline Dockerfile:14]: Do we want to use `scratch` and link statically?

File: deploy/operator.yaml (hunk 2 above)
Comment (pjbgf) [reply deploy/operator.yaml]: Very good point, just changed it to DaemonSet.

File: Dockerfile (hunk 3 above)
Comment (pjbgf) [reply Dockerfile:14]: I had some issues running from scratch, so was thinking on optimising (and also locking it down) once we have some the MVP functionality done and some automated E2E. What do you think?

Comment (saschagrunert) [reply Dockerfile:14]: Sounds good, let's leave it like it is for now.

**Pull request merged**